init.lua

```diff
@@ -32,20 +32,16 @@ function toolranks.create_description(name, uses, level)
   return newdesc
 end
 
+function toolranks.log2(value)
+  return math.log(value) / math.log(2)
+end
+
 function toolranks.get_level(uses)
-  if uses <= 100 then
```

Comment:
That's a pity, because I find playing with unlimited levels a lot more attractive. Would it be an option to add a setting for the maximum level that defaults to 6?